### DISCUSSION TITLE
Pattern Matching Values?

### DISCUSSION THREAD

**Discussion by geekox86** [gcanti/fp-ts#1728] [Q&A] [answered]
Greetings,

Thank you for this wonderful library!

How can I pattern match values like strings and numbers... etc.?

For example, in PureScript I can do:
toString 0 = "0"
toString x = "1"

Many thx.

**Comment by DenisFrezzato** [comment, marked as answer]
I don't think what you're asking falls within the scope of this library. I suggest you to take a look at https://github.com/gvergnaud/ts-pattern.
